domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt

```diff
@@ -77,6 +79,33 @@ object InteractionObjectTestBuilder {
     ).build()
   }
 
+  /** Creates fraction part for [NumberWithUnits].*/
```

Comment:
Missing space after `.`

`/** Creates fraction part for [NumberWithUnits]. */`

Reply:
Done